src/util/metrics.cpp

```diff
@@ -0,0 +1,44 @@
+
+#include <stdint.h>
+
+// TODO: Clean up Arduino code from this file
+#include <Arduino.h>
+
+#include "../util/utils.h"
+#include "../util/metrics.h"
+
```

Comment:
We may also need `metricsInit(struct metrics* metrics)` because `uint32_t sum` and `uint32_t current` are not guaranteed to be initialized to zero if `struct metrics` ever gets declared as a non-global variable.

Reply:
Very true, I missed adding sum into `metricsReset` like it should be when I refactored how averaging worked, will fix